recipes/pyxel-sim/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pyxel-sim" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxel-sim-{{ version }}.tar.gz
+  sha256: 8afdb9d31d9552908bb1256abca5983aa88a9b541901b3c438bebbc745a7c282
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyxel = pyxel.run:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=39.2
+    - wheel >=0.29.0
+  run:
+    - astropy >=4
+    - bokeh
+    - dask-core
+    - fsspec
+    - h5py
+    - holoviews
+    - ipywidgets >=7.5
+    - jupyterlab
+    - matplotlib-base >=3.1
+    - notebook
+    - numba
+    - numpy >=1.17
+    - openpyxl
+    - pandas
+    - pillow
+    - pympler
+    - python >=3.7
+    - pyyaml
+    - scikit-image
+    - scipy
+    - seaborn
+    - tqdm
+    - typing-extensions
+    - xarray
+    - xlrd
+    - pygmo
+test:
+  imports:
+    - pyxel
+    - pyxel.calibration
+  commands:
+    - pip check
+    - pyxel --help
+  requires:
+    - pip
+
+about:
+  home: https://esa.gitlab.io/pyxel/
+  summary: Pyxel detector simulation framework.
+  dev_url: https://gitlab.com/esa/pyxel
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - FredericLemmel
```

Comment:
you need to use your _github_username here (I guess `flemmel`?)

Looks good otherwise!

Reply:
You're right ! 

Now, I am using my GitHub username for the 'recipe-maintainers'.

Thanks !